docs/python_api/datasource/finngen/_finngen.md

```diff
@@ -12,4 +12,18 @@ title: FinnGen
   }
 </style>
 
-FinnGen is a research project in genomics and personalized medicine. It is large public-private partnership that has collected and analysed genome and health data from 500,000 Finnish biobank donors to understand the genetic basis of diseases. FinnGen is a now expanding into understanding the progression and biological mechanisms of diseases. FinnGen provides a world-class resource for further breakthroughs in disease prevention, diagnosis, and treatment and a outlook into our genetic make-up.
+[FinnGen](https://www.finngen.fi/en) is a research project in genomics and personalized medicine, representing a large public-private partnership. The project has collected and analyzed genome and health data from 500,000 Finnish biobank donors to understand the genetic basis of diseases. FinnGen is now expanding its focus to comprehend the progression and biological mechanisms of diseases. This initiative provides a world-class resource for further breakthroughs in disease prevention, diagnosis, and treatment, offering insights into our genetic makeup.
+
+We utilize data from FinnGen R9 (May 11, 2023, release):
+
+- Total sample size: 377,277 (210,870 females and 166,407 males)
+
+- Total number of variants analyzed: 20,175,454 variants
+
+- Number of disease endpoints (phenotypes) available: 2,272 endpoints
```

Comment:
I also have a feeling this documentation is more relevant for the actual data than the pipeline.

Reply:
What do you mean?